tests/characterization/commands/test_init_command.py

```
@@ -140,89 +140,87 @@ async def mock_process_request(text, state_manager, output=True):
     from tunacode.cli.commands import InitCommand
+
     command = InitCommand()
-    
+
     # Act
     await command.execute([], context)
-    
+
     # Assert
     assert prompt_sent is not None
     assert "TUNACODE.md" in prompt_sent
```

Pylint (3.3.7)

[error] Value 'prompt_sent' doesn't support membership test

(E1135)